content/docs/studio/rest-api.md

```diff
@@ -103,6 +103,143 @@ Running this example will fail because it requires that you have [set up remote
 cloud credentials] in DVC Studio to where the model is stored.
 
 </admon>
+</toggle>
+
+## Create new project
+
+To create a new project programmatically, you can integrate with the DVC Studio
+API. The API provides a POST request endpoint for creating projects.
+Authentication is required using a token, which can be generated from the user's
+profile settings in DVC Studio
+
+```yaml
+Endpoint: api/create-project
+HTTP Method: POST
+Content-Type: application/json
+```
+
+### Request
+
+Here are the possible values for each parameter in the request json:
+
+- `repo_url` (required): The URL of the repository where the project will be
+  created. This field is mandatory.
+- `public` (boolean, default: false): Specifies whether the project should be
+  public (`true`) or not (`false`).
+- `team_name`(string, default: none): The name of the team associated with the
+  project if we want to create the project in team space. This field is
+  optional. If team name is not specified, the project will be in user
+  namespace.
+- `subdir`(string, default: none): The subdirectory within the repository where
+  the project will be located, in case of a monorepo. This field is optional.
+- `name` (string, default: none): The name of the project to be defined. If no
+  name is specified, it will be extracted from repository name. This field is
+  optional.
+
+| header        | desc            | example value |
+| ------------- | --------------- | ------------- |
+| Authorization | Header for auth | token abc123  |
+
+Accepted formats for repository urls are:
+
+- `git@github.com:amritghimire/example-dvc-experiments.git`
+- `https://github.com/nimdraugsael/example-dvc-experiments.git`
+- `https://gitlab.example.org/amritghimire/example.git` (For custom gitlab)
+- `ssh://git@gitlab.com/xxx/yyy/zzz/repo.git`
+
+Similar format is supported for Github, Bitbucket and Gitlab.
+
+### Response
+
+Here is an example JSON response from the API request:
+
+```json
+{
+  "projects": [
+    {
+      "id": "tsanhawn9y",
+      "url": "https://studio.iterative.ai/user/amritghimire/projects/example-get-started-tsanhawn9y",
+      "name": "example-get-started"
+    }
+  ]
+}
+```
+
+This response confirms that the project was successfully created and provides
+the necessary information about the created project:
+
+- `id`: The unique identifier of the project. In this case, the project ID is
+  "tsanhawn9y".
+- `url`: The URL of the created project. You can access the project using this
+  URL. In this example, the project URL is
+  `https://studio.iterative.ai/user/amritghimire/projects/example-get-started-tsanhawn9y`.
+- `name`: The name of the created project. In this example, the project name is
+  "example-get-started".
+
+This information can be used to access and reference the project in the future.
+
+### Example
+
+First, you need your [DVC Studio access token]. For this example, we set it in
```

Comment:
link is broken, check other links on the page please

@yathomasi does our link checker suppose to catch it?

Reply:
Right now, it's just a string. There is no link associated so link checker is unable to detect this.

<img width="703" alt="Screenshot 2024-02-13 at 10 52 46" src="https://github.com/iterative/dvc.org/assets/20840228/aae9c3c3-5916-4ecc-aa56-aefd5f233eff">

Reply:
yep, I have seen it so many times recently that I'm starting thinking that we should probably detect [something] w/o a link attached to it

Reply:
It was when rebasing the branch, the name of the link seems to be changed. Fixed it now.